packages/tencent_cloud/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.1.1"
```

```suggestion
- version: "0.2.0"
```

Since its enhancement and not bugfix.